ci/code_checks.sh

```diff
@@ -100,6 +100,15 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
     cpplint --quiet --extensions=c,h --headers=h --recursive --filter=-readability/casting,-runtime/int,-build/include_subdir pandas/_libs/src/*.h pandas/_libs/src/parser pandas/_libs/ujson pandas/_libs/tslibs/src/datetime pandas/_libs/*.cpp
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # Get the output format for azure/githubaction from datapythonista
+    MSG='Check for use of not concatenated strings' ; echo $MSG
+    if [[ "$GITHUB_ACTIONS" == "true" ]]; then
+        $BASE_DIR/scripts/validate_string_concatenation.py .
```

Comment:
```suggestion
        $BASE_DIR/scripts/validate_string_concatenation.py . --format=""[error]{source_path}:{line_number}:\t BETWEEN {start} AND {end}"
```

This needs to have the format to be highlighted in GitHub actions

Reply:
Thank you :)